### PR TITLE
LPS-128485 Fix dialog footer position

### PR DESCRIPTION
**Pull request by ramonmsp**

### CODE REVIEW

**Comment by ramonmsp** [comment]
ci:test:sf

**Comment by ramonmsp** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
ci:test:relevant

File: modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/css/main.scss [hunk 1]
```diff
@@ -1020,3 +1020,9 @@
 		margin-top: 20px;
 	}
 }
+
```
Comment (rafaprax) [inline modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/css/main.scss:1023]: Hey @ramonmsp, Can you add some specific calendar class to make the selector specific for the calendar portlet?

Just to make sure that this change will only affect the calendar portlet when we use the it together with others portlets on the same page

Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

File: modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/css/main.scss (hunk 1 above)
Comment (ramonmsp) [reply modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/css/main.scss:1023]: I added one more class (calendar-portlet). It's enought?

**Comment by ramonmsp** [comment]
ci:test:sf

**Comment by ramonmsp** [comment]
ci:test:relevant

File: modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/css/main.scss (hunk 1 above)
Comment (rafaprax) [reply modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/css/main.scss:1023]: yeap

**Comment by ramonmsp** [comment]
ci:test:relevant

**Review by rafaprax — APPROVED** [review]

**Comment by ramonmsp** [comment]
ci:forward

**Pull request closed without merging**